tests/resources/deserializers/test_rocrate.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Module tests."""
+
+
+def test_rocrate_content_type(running_app, client_with_login, headers, search_clear):
```

Comment:
minor: Should we test the output of this against the "load" of the main RDM metadata schema?

Reply:
This is going through the service's RDM metadata schema.

Actually, this test is probably not correctly placed, since it's not only testing the deserializer class, but does an "E2E" call via the test client. I can move it higher up in the resources tree.